scripts/ci/run_perftest.sh

```diff
@@ -90,7 +94,10 @@ for scale in 50 100 500; do
   done
 done
 
-telegraf --once
+echo "Using Telegraph to report results from the following files:"
+ls $working_dir
+
+telegraf --debug --once
```

Comment:
Is this debugging that should be removed? Or something valuable to keep in the logs? (Not sure what the output looks like)

Reply:
no, I enabled debugging specifically to get more verbose log output from telegraf, which is very terse otherwise. My feeling was that, since the hosts will be terminated by the time these logs are ever accessed, more log content is best.

Reply:
This was after I found I had a specific question about what telegraf was doing that wasn't well-illuminated by Telegraf's default output.